plugin/src/App/Notifications.lua

```diff
@@ -129,46 +129,46 @@ function Notification:render()
 				}, {
 					Logo = e("ImageLabel", {
 						ImageTransparency = transparency,
-						Image = Assets.Images.PluginButton,
+						Image = if self.props.thirdParty then Assets.Images.ThirdPartyPlugin else Assets.Images.PluginButton,
 						BackgroundTransparency = 1,
 						Size = UDim2.new(0, 32, 0, 32),
 						Position = UDim2.new(0, 0, 0.5, 0),
 						AnchorPoint = Vector2.new(0, 0.5),
 					}),
-					Info = e("TextLabel", {
+					Message = e("TextLabel", {
 						Text = self.props.text,
-						Font = Enum.Font.GothamSemibold,
+						Font = Enum.Font.GothamMedium,
 						TextSize = 15,
 						TextColor3 = theme.Notification.InfoColor,
 						TextTransparency = transparency,
 						TextXAlignment = Enum.TextXAlignment.Left,
 						TextWrapped = true,
 
 						Size = UDim2.new(0, textBounds.X, 0, textBounds.Y),
-						Position = UDim2.fromOffset(35, 0),
+						Position = UDim2.fromOffset(38, 0),
 
 						LayoutOrder = 1,
 						BackgroundTransparency = 1,
 					}),
-					Time = e("TextLabel", {
-						Text = time:FormatLocalTime("LTS", "en-us"),
-						Font = Enum.Font.Code,
+					Info = e("TextLabel", {
+						Text = if self.props.source then "From: " .. self.props.source else time:FormatLocalTime("LTS", "en-us"),
```

Comment:
It's still worth keeping the time, imo:
```suggestion
						Text = (if self.props.source then "From: " .. self.props.source.. " •" else "").. time:FormatLocalTime("LTS", "en-us"),
```